src/modules/NetworkController/public/Get-SdnNetworkControllerNode.ps1

```diff
@@ -39,11 +39,19 @@ function Get-SdnNetworkControllerNode {
             }
         }
 
-        if (Test-ComputerNameIsLocal -ComputerName $NetworkController) {
-            $result = Get-NetworkControllerNode
+        try {
+            if (Test-ComputerNameIsLocal -ComputerName $NetworkController) {
+                $result = Get-NetworkControllerNode -Name $NetworkController -ErrorAction Stop
```

Comment:
`Get-SdnInfrastructureInfo` called `Get-SdnNetworkControllerNode` to get the list of Network Controller Nodes expect an array of NC Node. 
`Get-SdnNetworkControllerNodeCertificate` called the same but expect single node (current NC) returned. 

We might need to add more parameter like

- -Server to filter the NC node by Server Name, short or FQDN.
- -NodeName to filter by NC Node name
@arudell